### ISSUE TITLE
Multi-site support

### ISSUE THREAD

**Issue by jessedobbelaere** [shornuk/craft-plausible#20]
👋 I have a Craft multi-site website. So there are two different sites for the same company, each site has its own domain name. I configured two Plausible "sites" for tracking. 

In SEOMatic, you can enter a different Plausible tracking script per site:
![Screenshot 2024-04-11 at 10 06 42@2x](https://github.com/shornuk/craft-plausible/assets/1352979/0fd90544-adb3-4b1c-9ec1-c13e00db6c39)

The Plausible plugin on Craft CMS only supports one `Site ID`. **Would it be interesting to allow to configure multiple? So the Craft dashboard can show widgets from both sites?**

**Comment by Mathias-Syversen** [comment]
This would be very welcome, we use a lot of multisites for marketing and miss beeing able to use the widgets per site.

**Comment by wmdhosting** [comment]
This will be great .. i have 130 site install..  Town portals.. 

Use SEOmatic too.. 

Is there perhaps some way   when multi site...  

To pull site ID from Seomatic if easier..  to do .. than refactoring plugin.. for multisite.. 

I can  sponsor  this upgrade.. ..

**Comment by AndrewMac** [comment]
Yes, please. Perhaps the widget could pick up the Site ID from Plausible, rather than the generic settings page? (In my case, at least, the API key could be generic.)

It would be wonderful if you could add this! Cheers, Andrew

**Comment by AndrewMac** [comment]
Yes, please. Perhaps the widget could pick up the Site ID from Plausible, rather than the generic settings page? (In my case, at least, the API key could be generic.)

If you could add this! Cheers, Andrew
